packages/office-ui-fabric-react/src/components/Checkbox/Checkbox.base.tsx

```diff
@@ -179,24 +170,25 @@ export class CheckboxBase extends React.Component<ICheckboxProps, ICheckboxState
   };
 
   private _onChange = (ev: React.FormEvent<HTMLElement>): void => {
-    const { disabled, onChange } = this.props;
+    const { onChange } = this.props;
```

Comment:
This event shouldn't be getting called to begin with if the component is disabled (other form components assume this), so the function doesn't need to include a disabled check.